test/runtests.jl

```diff
@@ -301,12 +343,25 @@ end
     @test_throws MethodError DataAPI.colmetadatakeys(1, 1)
     @test_throws MethodError DataAPI.colmetadatakeys(1)
 
+    dim = 1
```

Comment:
line not needed?

Reply:
Ah - now I see how you use it. But why do you set it twice then and do I understand it correctly that it is always `1`?

Reply:
Why not write `1` instead of using a variable? That sounds clearer to me.

Reply:
> The column metadata tests use `:col` as the argument for which column is being accessed. I thought using the variable `dim` instead of just `1` would make it easier to quickly read and understand what exactly was being tested.

@Tokazama I'm not convinced using `dim` is clearer, as you have to check what `dim` corresponds to. Also as you note the other tests using `:col` repeat it for each line so that's not consistent.

Reply:
If you think replacing it with '1' will improve readability I'm happy to change it

Reply:
Please do.